src/mercury_engine_data_structures/formats/bmsld.py

```diff
@@ -166,7 +166,7 @@ def get_actor_group(self, group_name: str) -> Container:
             None
         )
         if group is None:
-            raise Exception(f"No group found with name for {group_name}")
+            raise ValueError(f"No group found with name for {group_name}")
```

Comment:
tbh I'd prob call this a KeyError, even if it isn't technically a dictionary. @henriquegemignani thoughts?

Reply:
Changed it to KeyError now.  Both are kinda matching / non-matching. It's not a dict but very similar lookup. Otherwise you could say, you just used a wrong value for the parameter.

(just forced push as the last two changes were pretty small)

Reply:
It's a container-like enough for me.